doc/api/assert.md

```diff
@@ -147,6 +147,114 @@ try {
 }
 ```
 
+## Class: assert.CallTracker
+
+### `new assert.CallTracker()`
+<!-- YAML
+added: REPLACEME
+-->
+
+Returns a [`CallTracker`][] object.
+
+```js
+const assert = require('assert');
+
+const tracker = new assert.CallTracker();
+
+function func() {}
+
+const callsfunc = tracker.calls(func);
```

Comment:
Nit:
```suggestion
const callsfunc = tracker.calls(func, 1);
```
To be as clear as possible, the default values can be understood from the `calls` doc itself imo.